{"name": "Проверка создания двух одинаковых курьеров", "status": "passed", "description": "Создаем курьера с существующим логином. Получаем сообщение об ошибке \"Этот логин уже используется. Попробуйте другой.\" и код 409", "start": 1718117971744, "stop": 1718117971831, "uuid": "51700624-6837-43da-a184-29737b1225a0", "historyId": "7434085135fa2a0d3218fdb141b0e3c2", "testCaseId": "7434085135fa2a0d3218fdb141b0e3c2", "fullName": "tests.test_create_courier.TestCreateCourier#test_create_courier_double", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "MBAirDmitry.local"}, {"name": "thread", "value": "81946-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_courier"}]}